lib/metadataTypes/Email.js

```diff
@@ -15,14 +15,29 @@ class Email extends MetadataType {
      * Retrieves SOAP based metadata of metadata type into local filesystem. executes callback with retrieved metadata
      *
      * @param {string} retrieveDir Directory where retrieved metadata directory will be saved
+     * @param {void} [_] unused parameter
+     * @param {void} [__] unused parameter
+     * @param {void} [___] unused parameter
+     * @param {string} [key] customer key of single item to retrieve
      * @returns {Promise.<TYPE.MetadataTypeMapObj>} Promise of metadata
      */
-    static retrieve(retrieveDir) {
+    static retrieve(retrieveDir, _, __, ___, key) {
         Util.logger.warn(
             '- Classic E-Mails are deprecated and will be discontinued by SFMC in the near future. Ensure that you migrate any existing E-Mails to Content Builder as soon as possible.'
         );
+        /** @type {TYPE.SoapRequestParams} */
+        let requestParams = null;
+        if (key) {
```

Comment:
stylistic thing - I would prefer that in these cases we put it right in function as a ternary operator (key? {OBJECT} : null)  but i can see in some cases it may be more messy so its more a consideration if you agree

Reply:
normally i'd agree but given the multiple complicated cases i'd rather have it the same way across all files to keep readability up